assets/react/v3/shared/utils/util.ts

```diff
@@ -256,20 +256,20 @@ export const parseNumberOnly = (value: string) => {
 };
 
 export const throttle = <T extends (args: MouseEvent) => void>(func: T, limit: number) => {
-	let inThrottle = false;
-
-	return function (this: ThisParameterType<T>, ...args: Parameters<T>) {
-		if (!inThrottle) {
-			func.apply(this, args);
-			inThrottle = true;
-
-			setTimeout(() => {
-				inThrottle = false;
-			}, limit);
-		}
-	};
+  let inThrottle = false;
```

Comment:
I don't find any usability of this variable.

Reply:
`inThrottle` is used inside the return function